backend/emailSystem.py

```diff
@@ -0,0 +1,34 @@
+import smtplib, ssl
+
+port = 465 #For SSL
+password = ""#input("password: ") #we can add the actual password here instead
+
+# Create a secure SSL context
+context = ssl.create_default_context()
+
+#EMAIL
+sender_email = "goldchest.steam@gmail.com" #our email needs to be changed
+reciever_email = "samuel.jothimuthu@gmail.com" #sender email will need to be updated
+message_test = """\
+Subject: Steam Recommendations
+
+Hello World!
+This is a test message. 2
+
+"""
+message_newUser = """\
```

Comment:
Maybe mention the username of the person who joined

Reply:
Isn't the username the email address, so wouldn't they already have it if the email is going to their email address

Reply:
Right, forgot about that.